modules/tinymce/src/themes/silver/main/ts/ui/general/OuterContainer.ts

```diff
@@ -81,9 +81,9 @@ interface OuterContainerApis {
 
 interface ToolbarApis {
   readonly setGroups: (toolbar: AlloyComponent, groups: SketchSpec[]) => void;
-  readonly refresh: (toolbar: AlloyComponent) => void;
-  readonly toggle: (toolbar: AlloyComponent) => void;
-  readonly toggleWithoutFocusing: (toolbar: AlloyComponent) => void;
+  readonly refresh?: (toolbar: AlloyComponent) => void;
+  readonly toggle?: (toolbar: AlloyComponent) => void;
+  readonly toggleWithoutFocusing?: (toolbar: AlloyComponent) => void;
```

Comment:
I prefer setting refresh to `Fun.noop` to making the methods optional

- [ ] revert the last commit

Reply:
This type is wrong and all the other calls to Toolbar apis is wrapped in that mapFrom null check. I think we should be fine with this change.

Reply:
But I guess we could do it two stage. Revert this then make another PR that we marge after the release that does this proper.

Reply:
done